### PR TITLE
skip increasing counters

### PR DESCRIPTION
**Pull request by krlosMata**
This PR does the following:
- Add  flag `--nc` to skip counters increments
- Skip incrementing counters during execution

### CODE REVIEW

File: src/sm/sm_main/sm_main_exec.js [hunk 1]
```diff
@@ -280,8 +290,13 @@ module.exports = async function execute(pols, input, rom, config = {}) {
         }
 
         if (l.inSTEP) {
-            op0 = Fr.add(op0, Fr.mul( Fr.e(l.inSTEP), Fr.e(i)));
-            pols.inSTEP[i] = Fr.e(l.inSTEP);
+            if (skipCounters) {
+                op0 = 1n;
```
Comment (zkronos73) [inline src/sm/sm_main/sm_main_exec.js:294]: I known that skipCounters was a exceptional situation, but for me was better: 
op0 = Fr.add(Fr.one, Fr.mul( Fr.e(l.inSTEP), Fr.e(i)));

File: src/main_executor.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ const argv = require("yargs")
     .alias("e", "execute")
     .alias("v", "verbose")
     .alias("T", "tracer")
+    .alias("nc", "nocounters")
```
Comment (zkronos73) [inline src/main_executor.js:44]: IMO would have been better to use one letter, no two letters.

**Review by zkronos73 — APPROVED** [review]

File: src/sm/sm_main/sm_main_exec.js (hunk 1 above)
Comment (krlosMata) [reply src/sm/sm_main/sm_main_exec.js]: I would maintain `Fr.one` just to increase the step

File: src/main_executor.js (hunk 2 above)
Comment (krlosMata) [reply src/main_executor.js]: agree! I have chamge it xD

**Pull request merged**